stdlib/ostruct.rb

```diff
@@ -60,10 +66,52 @@ def to_n
   def hash
     @table.hash
   end
+  
+  def table
+    @table
+  end
+  
+  def delete_field(name)
+    sym = name.to_sym
+    begin
+      singleton_class.__send__(:remove_method, sym, "#{sym}=")
+    rescue NameError
+    end
+    @table.delete sym
+  end
+  
+  def new_ostruct_member(name)
```

Comment:
I got that from empirical tests, don't trust the docs blindly, some methods may be coming in from the C side of things or god knows where.

```ruby
>> require 'ostruct'
=> true
>> OpenStruct.new.new_ostruct_member("test")
NoMethodError: undefined method `new_ostruct_member' for #<OpenStruct>
    (ripl):2:in `<main>'
>> 
```